docs/controls/MarkdownTextBlock.md

```diff
@@ -95,7 +96,7 @@ The MarkdownTextBlock control is highly customizable to blend with any theme. Cu
 
 ### LinkClicked
 
-Use this event to handle clicking on links for Markdown, by default the MarkdownTextBlock does not handle Clicking on Links.
+Use this event to handle clicking on links or images for Markdown, by default the MarkdownTextBlock does not handle Clicking on Links or Images.
```

Comment:
Could you also add a note here that taping any image will raise the LinkClicked event and with the uri of the image?Developers should use the LinkType property to decide how to handle it.

Reply:
Done. Let me know if there is anything else that i need to update.

Reply:
Thanks. @WilliamABradley up to you